services/mailer/src/integration-tests/bootstrap.ts

```diff
@@ -0,0 +1,59 @@
+import "mocha";
+import { AwilixContainer } from "awilix";
+import { Application } from "../app/application.types";
+import { createContainer } from "../container";
+import { appConfig } from "../config/config";
+import { createClient } from "redis";
+
+export interface BootstrapData {
+  app: Application;
+  container: AwilixContainer;
+  redisUrl: string;
+  redisPrefix: string;
+}
+
+export type GetBootstrap = () => BootstrapData;
+
+export interface GlobalData extends NodeJS.Global {
+  getBootstrap: GetBootstrap;
+  bootstrap: BootstrapData;
+}
+
+declare const global: GlobalData;
+
+before("Create container and dependencies", async function () {
+  const container = await createContainer(appConfig);
+  const app = container.resolve<Application>("app");
+  const redisUrl = container.resolve<string>("redisUrl");
+  const redisPrefix = container.resolve<string>("redisPrefix");
+
+  global.getBootstrap = () =>
+    Object.freeze({
+      app,
+      container,
+      redisUrl,
+      redisPrefix,
+    });
+});
+
+beforeEach("Clean email queue", async function () {
+  const { redisPrefix, redisUrl } = global.getBootstrap();
+
+  const redisClient = createClient({
+    url: redisUrl,
+    prefix: redisPrefix,
+  });
+
+  redisClient.flushall();
```

Comment:
FLUSHALL method deleting all keys from every databases. You can use the FLUSHDB method instead.

Reply:
@mkopa thanks for pointing that. I have changed it to `del()`